package.json

```diff
@@ -13,7 +13,7 @@
   "dependencies": {
     "@dashevo/dark-gravity-wave": "^1.1.1",
     "@dashevo/dash-util": "^2.0.3",
-    "@dashevo/dashcore-lib": "^0.19.29",
+    "@dashevo/dashcore-lib": "^0.19.39",
```

Comment:
Does it make sense to specify in both dependency sections?

Reply:
It has to be in both peerDependencies and devDependencies. peer dependency used by host projects, and dev dependency is used for local development and tests